gui/middleware/zfs.py

```diff
@@ -668,6 +668,8 @@ def parse_status(name, doc, data):
             })
             scrub_status = 'IN_PROGRESS'
             scrub_statusv = _('In Progress')
+            if scan.find('scrub paused') != -1:
+                scrub_statusv = _('Paused')
```

Comment:
Sounds like we should also set `scrub_status` and a few places we need special handling of this status, e.g. volume status datagrid, boot pool status and scrub start/stop.